### DISCUSSION TITLE
Loading of project fails

### DISCUSSION THREAD

**Discussion by anderskiaer** [community/community#62524] [Mobile] [answered]
### Select Topic Area

Bug

### Body

Initially, when support for GitHub projects where added to the mobile app, it worked as expected.

However now, over some months, it has consistently failed when trying to view the project in the mobile application. It does not give any clues on what actually fails. Viewing the project on github.com through web browser works as expected. 🤔 

![IMG_3668](https://github.com/community/community/assets/9577823/bb90d68e-d949-4ad7-90b1-771a409823df)

_Version: 1.124.0 of the GitHub app on iOS._

**Comment by mxie** [comment]
I'm sorry to hear you're experiencing issues! To start off, can you tell us how large the project is (how many total items expected)?

**Reply by anderskiaer** [reply]
Thanks for your reply @mxie. The unfiltered project is currently quite small (number of items ~100). 

I did some debugging and looks like a date filter is causing problems for the GitHub mobile application. If I remove the filter `deadline:>=@today-7` (`deadline` being a date property on the issues in the project) it works also in the mobile application. Adding the filter back makes the problem reappear in the mobile application.

So I guess the issue I've encountered here is that the mobile application does not (yet?) support all filters listed in https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/filtering-projects#filtering-iteration-and-date-fields-using-keywords?

**Comment by anderskiaer** [comment, marked as answer]
This now appears to be fixed in recent version of the GitHub app 👍🏻

**Comment by Soner05** [comment]
Kimmm
